.github/workflows/android-perf.yml

```diff
@@ -277,12 +277,8 @@ jobs:
       project-arn: arn:aws:devicefarm:us-west-2:308535385114:project:02a2cf0f-6d9b-45ee-ba1a-a086587469e6
       device-pool-arn: ${{ matrix.device }}
       # Uploaded to S3 from the previous job, the name of the app comes from the project itself.
-      # Unlike models there are limited numbers of build flavor for apps, and the model controls whether it should build with bpe/tiktoken tokenizer.
-      # It's okay to build all possible apps with all possible flavors in job "build-llm-demo". However, in this job, once a model is given, there is only
-      # one app+flavor that could load and run the model.
-      # TODO: Hard code llm_demo_bpe for now in this job.
-      android-app-archive: https://gha-artifacts.s3.amazonaws.com/${{ github.repository }}/${{ github.run_id }}/artifact/llm_demo_bpe/app-debug.apk
-      android-test-archive: https://gha-artifacts.s3.amazonaws.com/${{ github.repository }}/${{ github.run_id }}/artifact/llm_demo_bpe/app-debug-androidTest.apk
+      android-app-archive: https://gha-artifacts.s3.amazonaws.com/${{ github.repository }}/${{ github.run_id }}/artifact/llm_demo/app-debug.apk
+      android-test-archive: https://gha-artifacts.s3.amazonaws.com/${{ github.repository }}/${{ github.run_id }}/artifact/llm_demo/app-debug-androidTest.apk
```

Comment:
If the new app doesn't cover llm models, the workflow needs to be updated accordingly. Otherwise you may end up having a non-genai model deployed with llm app, and vice versa

Reply:
That can be done later. For now, let's verify if the new app is able to load and run nongenai models (for xnnpack and qnn). The app needs to be built with both xnnpack and qnn which is fine since we are benchmarking the model instead of the app. For running the QNN model the device model/serial may be an issue per https://github.com/pytorch/executorch/issues/4973, if yes, I think we should at least make sure the app can built with qnn delegate successfully